CLOUD_TPU_README.md

```diff
@@ -0,0 +1,11 @@
+# Cloud TPU
+If you’re looking for samples for how to use Cloud TPU, check out the guides here. 
+
```

Comment:
will this file be moved under the `/tpu` directory as its root readme?

Note: we do have two tpu samples `/tpu/training` and `tpu/hptuning` now.  please add them here (or i can do it after your PR is merged)

Reply:
No, since these samples aren't actually using CMLE for TPU use, but rather are the docs pages tutorials for the TPU product. The idea was to move them away from the CMLE README but still link to this README in the main page README.

Reply:
The 2 TPU samples are linked in the main README.

Reply:
Can we then make this file more generic rather than being TPU-specific? This way we can add more material from different related topics

Reply:
Note of offline discussion:
- Add new additional Resources section
- Leave this README for TPU
- Example: when new README for community work is added, create link in Additional Resources
- If Additional Resources sections becomes too large, create a more general README to hold this information.